src/fides/cli/__init__.py

```diff
@@ -40,7 +40,7 @@
 API_COMMAND_DICT = {command.name or str(command): command for command in API_COMMANDS}
 ALL_COMMANDS = API_COMMANDS + LOCAL_COMMANDS
 SERVER_CHECK_COMMAND_NAMES = [
-    command.name for command in API_COMMANDS if command.name not in ["status"]
+    command.name for command in API_COMMANDS if command.name not in ["status", "worker"]
```

Comment:
The worker needn't depend on the webserver to be running in order to run itself. Processing tasks from the Celery queues and adding tasks to the queues should operate independently of one another.

Reply:
good catch here! I need that opspertise 😏